src/Squirrel/UpdateManager.ApplyReleases.cs

```diff
@@ -62,7 +62,7 @@ public ApplyReleasesImpl(string rootAppDirectory)
                     return getDirectoryForRelease(updateInfo.CurrentlyInstalledVersion.Version).FullName;
                 }
 
-                var ret = await this.ErrorIfThrows(() => installPackageToAppDir(updateInfo, release), 
```

Comment:
All these trailing-space deletions are distracting, especially in a repo that's mainly someone else's.